twofactor_privacyidea/lib/Provider/TwoFactorPrivacyIDEAProvider.php

```diff
@@ -252,12 +251,23 @@ private function getClientOptions()
      */
     public function verifyChallenge(IUser $user, $challenge)
     {
+        return $this->authenticate($user->getUID(), $challenge);
+    }
+
+    /**
+     * Try to authenticate the given username with the given password.
+     *
+     * @param string $username
+     * @param string $challenge
+     * @return Boolean, True in case of success
+     */
+    public function authenticate($username, $challenge) {
```

Comment:
could we please at least change the misleading word ``$challenge`` here to ``$password``?

Reply:
Definitely :) d296de9 changes this.